.gitignore

```diff
@@ -6,10 +6,25 @@ bin/
 build/
 ci/
 
+# Gradle files
+.gradle/
+build/
+ci/
```

Comment:
These files are already included above on lines 5-7. I also don't think we should ignore `.gradle` files.

Reply:
Thanks for catching these @kevinlind .  Updated.